frontend/src/components/authentication/LoginForm.jsx

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import LOCALIZE from "../../text_resources";
+
+const styles = {
+  loginContent: {
+    padding: "12px 32px 0 32px"
+  },
+  inputTitles: {
+    padding: "12px 0 6px 0",
+    fontWeight: "bold"
+  },
+  inputs: {
+    width: "100%",
+    padding: "3px 6px 3px 6px",
+    border: "1px solid #00565E",
+    borderRadius: 4
+  },
+  loginBtn: {
+    width: 150,
+    display: "block",
+    margin: "24px auto"
+  },
+  validationError: {
+    color: "red",
+    paddingTop: 8
+  }
+};
+
+class LoginForm extends Component {
+  //TODO(fnormand): Remove this part when implementing login functionality in the backend
+  //===========================================
+  static propTypes = {
+    authentification: PropTypes.func
+  };
+
+  state = {
+    isAuthenticated: false
+  };
+
+  handleSubmit = () => {
+    this.setState({ isAuthenticated: true });
+    this.props.authentification();
+  };
+  //===========================================
+
+  render() {
+    return (
+      <div>
+        {!this.state.isAuthenticated && (
+          <div>
+            <div style={styles.loginContent}>
+              <h3>{LOCALIZE.homePage.login.content.title}</h3>
```

Comment:
See (Comment Ref 1)

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/152/commits/6ce81fd2552bf6633dd7149f12e43f64df4e3db2